javatests/google/registry/flows/domain/DomainCreateFlowTest.java

```diff
@@ -1195,6 +1196,15 @@ public void testFailure_premiumBlocked() throws Exception {
   }
 
   @Test
+  public void testFailure_extraBlocked() throws Exception {
+    thrown.expect(ExtraDomainError.class);
```

Comment:
Sure thing. Would you like me to also refactor the existing tests so they have the .expect() directly before the method that throws the exception?